### PR TITLE
Update the columns block to use the new colors hook

### PR DESCRIPTION
**Pull request by youknowriad**
Based on #21021 to try test the API on different blocks (columns for this PR)

### CODE REVIEW

File: packages/block-library/src/columns/style.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
 .wp-block-columns {
 	display: flex;
 	margin-bottom: $default-block-margin;
+	background-color: var(--wp--background-color);
```
Comment (gziolo) [inline packages/block-library/src/columns/style.scss]: This one feels safe to use 👍

**Review by gziolo — COMMENTED** [review]
Nice, I like the direction.

Unrelated note. The number of experimental props used for RichText and InnerBlocks makes it nearly impossible for 3rd party devs to follow code implementation uses in Gutenberg 🙁 It also means it’s all undocumented and difficult to use for contributors...

**Comment by youknowriad** [comment]
> The number of experimental props used for RichText and InnerBlocks makes it nearly impossible for 3rd party devs to follow code implementation uses in Gutenberg

True, we need to continue our audits. the color hook is something I want to stabilize pretty quickly

**Comment by youknowriad** [comment]
This one is ready to ship.

**Review by gziolo — APPROVED** [review]
Indeed it is, nice work 👍

**Comment by youknowriad** [comment]
Oh, it seems we might need a deprecated version here, I'll add it and ship.

File: packages/block-editor/src/components/block-list/block-wrapper.js [hunk 2]
```diff
@@ -101,7 +101,9 @@ const BlockComponent = forwardRef(
 			// should only consider tabbables within editable display, since it
 			// may be the wrapper itself or a side control which triggered the
 			// focus event, don't unnecessary transition to an inner tabbable.
-			if ( wrapper.current.contains( document.activeElement ) ) {
+			if (
+				isInsideRootBlock( wrapper.current, document.activeElement )
```
Comment (youknowriad) [inline packages/block-editor/src/components/block-list/block-wrapper.js]: For me this is a bug that is present on master too. If you want to select a block, you want to select the root block, not an inner one.

File: packages/e2e-tests/specs/editor/various/block-hierarchy-navigation.test.js [hunk 3]
```diff
@@ -91,7 +91,7 @@ describe( 'Navigating the block hierarchy', () => {
 		await pressKeyWithModifier( 'ctrl', '`' );
 		await pressKeyWithModifier( 'ctrl', '`' );
 		await pressKeyWithModifier( 'ctrl', '`' );
-		await pressKeyTimes( 'Tab', 4 );
+		await pressKeyTimes( 'Tab', 5 );
```
Comment (youknowriad) [inline packages/e2e-tests/specs/editor/various/block-hierarchy-navigation.test.js]: The colors panel is shown before the columns count now.

**Pull request merged**